stacks/core/dev/stage-1.yml

```diff
@@ -68,8 +72,9 @@ services:
       replicas: 1
       labels:
         - traefik.enable=true
-        - traefik.frontend.rule=HostRegexp:bt,bt.{domain:[a-zA-Z0-9.]+}
-        - traefik.port=8080
+        - traefik.http.services.babeltower.loadbalancer.server.port=8080
+        - traefik.http.routers.babeltower.rule=HostRegexp(`{subdomain:api}.{domain:[a-zA-Z0-9.]+}`)
```

Comment:
Please update the [docs](https://github.com/CESARBR/knot-cloud/tree/master/stacks/core#configure-dns) to reflect these changes :smile:

Reply:
Done.